UNIXFS.md

```diff
@@ -59,6 +75,31 @@ For files that are comprised of more than a single block, the 'Type' field will
 
 This data is serialized and placed inside the 'Data' field of the outer merkledag protobuf, which also contains the actual links to the child nodes of this object.
 
+For files comprised of a single block, the 'Type' field will be set to 'File', 'filesize' will be set to the total number of bytes in the file and the file data will be stored in the 'Data' field.
+
+The serialized size of a UnixFS node must not exceed 256KiB in order to work will with the [Bitswap] protocol.
```

Comment:
not quite: It can be up to 1MiB (bitswap will transfer 2 (4?) MiB blocks).

Reply:
I've removed this last paragraph, we can always add it in later when the bitswap spec is more fleshed out.